gateway-eth-ts/package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint src/ --ext .ts",
     "build": "yarn compile && yarn doc",
     "pretest": "cd ../smart-contract && (NODE_ENV=test yarn local &) && sleep 10",
-    "test": "ts-mocha src/**/*.test.ts",
+    "test-local": "ts-mocha test/**/*.test.ts",
```

Comment:
Verify if rename is needed to avoid hardhat collation.

Reply:
Yep the name collision is there. If the name is `test` then hardhat will try to spin up a node and run. Keeping the name change.